docs/source/sparse.rst

```diff
@@ -1021,7 +1156,10 @@ The following :mod:`torch` functions support sparse tensors:
 :func:`~torch.zeros`
 :func:`~torch.zeros_like`
 
-In addition, all zero-preserving unary functions support sparse
+Unary functions
+---------------
+
+All zero-preserving unary functions support sparse
```

Comment:
All zero-preserving unary functions *listed below* support sparse...

or possibly

All zero-preserving unary functions *will* support sparse...

Some zero-preserving unary functions, such as `reLU`, are not in fact supported, but the goal is to make sure they all are, so we might want the wording here to reflect that any missing zero-preserving unary function support is a reportable bug. 

I also do not think this list is completely correct: `torch.angle` does not appear to support COO. Also all layouts support `torch.expm1` which is missing.

Reply:
I'll update the wording to say that all zero preserving unary functions *can* be supported and encourage the reader to open an issue if they find one that's missing. I think it's fair to say that they're pretty easily supported, so it's ok to encourage people to report it.

Reply:
That sounds great. It is easy to add them so we should encourage people to point out missing ones.